docs/agent/version-support-policy.mdx

```diff
@@ -44,7 +44,8 @@ Extended support beyond LTS is available to customers on a commercial basis. It
 
 | Support Status | Release | Release Date | Latest | Active Support | End of Life |
 | -------------- | ------- | ------------ | ------ | -------------- | ----------- |
-| Active         | 3.23x   | 2025-06-05   | 3.23.3 | 2026-06-05     | 2026-08-05  |
+| Active         | 3.24x   | 2025-07-17   | 3.24.0 | 2026-07-17     | 2026-09-17  |
+| Active         | 3.23x   | 2025-06-05   | 3.23.3 | 2026-06-20     | 2026-08-05  |
```

Comment:
```suggestion
| Active         | 3.23x   | 2025-06-26   | 3.23.3 | 2026-06-26     | 2026-08-26  |
```

should this match the release date in changeling.md?

Reply:
switched release date back to 06/05, but I see what you mean on the interpretation of that paragraph could go either way, feel free to ignore the suggestion!